recipes/sqlite3/all/conanfile.py

```diff
@@ -86,6 +89,10 @@ def configure(self):
         del self.settings.compiler.libcxx
         del self.settings.compiler.cppstd
 
+        if (not self.options.enable_default_vfs) and self.options.build_executable:
+            # Need to provide custom VFS code: https://www.sqlite.org/custombuild.html
+            raise ConanInvalidConfiguration("Building executable is not supported without default OS interface")
```

Comment:
I found the original wording confusing.
```suggestion
            raise ConanInvalidConfiguration("build_executable=True cannot be combined with enable_default_vfs=False")
```

Reply:
Thanks, that was fixed